conf/i18n/en/my-account.yaml

```diff
@@ -0,0 +1,66 @@
+myAccount:
```

Comment:
Maybe that I'm not German ;)
Anyway, in here it makes no sense to provide the German version where nobody can properly edit it. Once it is in the theme, then I will just ask anybody to translate it. The changes I do in the app are just to make it work.